smaug/python/graph.py

```diff
@@ -187,9 +184,6 @@ def print_summary(self):
       print("Parents:", end = '')
       for i in node.parents:
         print(i, end = ' ')
-      print("\nChildren:", end = '')
```

Comment:
IMO this info is actually pretty useful, even if it is redundant.

Reply:
Yeah, it's useful but, if it means a lot more work, I'd remove this until we really want to add it back.